### DISCUSSION TITLE
ente auth does not sync

### DISCUSSION THREAD

**Discussion by larscrismes** [ente-io/ente#3833] [Q&A]
I added a new OTP via my mobile.
Worked fine. But my other two devices (windows, android) do not get it synced.

**Comment by crKRoEYS** [comment]
Are you logged in with Ente Account? Which active sessions do you see in Auth settings? Also please verify that the time on all your devices is the same.

**Reply by larscrismes** [reply]
Thanks for your support. Yes both other devices are logged in to the same a/c.
Weired, meanwhile the windows device has synced it. Also this device shows active sessions.
The other device can`t load the session page (cancel after certain time) and also fails to log out for some reason...?!?!

**Reply by larscrismes** [reply]
strange behavior on the android 13 device. The active sessions on other device dates it back to March.
Logout was not possible. I deleted cache and data, but then no login was possible.
Seems something broke the app. Reinstalled it and now all OTP are available...

**Reply by crKRoEYS** [reply]
So all devices synchronize now?

**Comment by larscrismes** [comment]
yepp, meanwhile the two renegades are back working well, but I think of two different reasons.
The Android device has broken app and need to be reinstalled. Clear action.
The windows device first did not show the new code, but after a while it worked. Without any action.
So I don`t know what was the issue...

**Comment by Erw1nn** [comment]
I'm having a very similar issue with my codes not syncing. I have a Galaxy S25 that I just imported my all my 2FA codes into an hour ago and seems to be fine but when I try to open the app on my Galaxy Tab 8, none of my codes aren't there
